packages/components/src/modifiers/hds-clipboard.ts

```diff
@@ -40,7 +41,7 @@ export const getTargetElement = (target) => {
       return;
     }
   } else if (target instanceof Node && target.nodeType === Node.ELEMENT_NODE) {
-    targetElement = target;
+    targetElement = target as Node;
```

Comment:
we guard for the type on L43, so there's no need to use `as Node` here
```suggestion
    targetElement = target;
```